Formula/grammarly-languageserver.rb

```diff
@@ -0,0 +1,61 @@
+class GrammarlyLanguageserver < Formula
+  desc "This is a language server implementation of Grammarly"
+  homepage "https://github.com/znck/grammarly"
+  url "https://github.com/znck/grammarly/archive/refs/tags/v0.22.1.tar.gz"
+  sha256 "749e2577f7c04aa9a80a39e7e8cf619e7f7e85341d3a52b9c8174b3295e2e264"
+  license "MIT"
+
+  depends_on "pnpm" => :build
+  depends_on "node@16"
```

Comment:
Does it not work with a modern node?

Reply:
Node@16 is required by the project in their `package.json`: https://github.com/znck/grammarly/blob/main/package.json#L4

In fact, it seems to work on the latest node LTS version, i.e., v18, except that there will be warnings saying the current node version mismatches the version required by the program. 
But to be safe from unexpected hidden failures, I stick it to node@16.

Reply:
This is no longer a problem since I rewrote the formula to use the npm package published by the upstream.